azure-devops/azext_devops/dev/pipelines/arguments.py

```diff
@@ -34,12 +34,6 @@ def load_build_arguments(self, _):
         context.argument('definition_id', type=int)
         context.argument('variables', nargs='*')
 
-    with self.argument_context('pipelines build show') as context:
-        context.argument('build_id', options_list=('--build-id', '--id'), type=int)
-
-    with self.argument_context('pipelines build definition show') as context:
-        context.argument('definition_id', options_list=('--definition-id', '--id'), type=int)
-
```

Comment:
Couldn't comment at unchanged line number. 
task_id should be renamed to 'id' as well